src/components/incentives/IncentivesButton.tsx

```diff
@@ -61,6 +63,35 @@ export const MeritIncentivesButton = (params: {
   );
 };
 
+export const ZkIgniteIncentivesButton = (params: {
+  market: string;
+  asset?: string;
```

Comment:
I see we are passing the term asset around. Can we make it more explicit like rewardAsset or something similar. This way its easier to understand when we are passing this variable @MartinGbz

Reply:
Agree with you
Asset here can be the aToken or the vToken (aave asset in any cases). If it was the underlying asset I would set it for `underlyingAsset`.
I can set it to `rewardedAsset` what do you think?

Reply:
Pushed a commit: https://github.com/aave/interface/pull/2305/commits/f2dad18f344cf6e907b8d7af53c30147de67c168